### PR TITLE
improvement: BB-57 use updated probe server

### PR DESCRIPTION
**Pull request by naren-rajendran**
Use updated probe server in ingestion processes.

### CODE REVIEW

File: extensions/lifecycle/conductor/service.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ function handleMetrics(res, log) {
 }
 
 async.waterfall([
-    done => startProbeServer(config.healthcheckServer, (err, probeServer) => {
+    done => startProbeServer(lcConfig.conductor.probeServer, (err, probeServer) => {
```
Comment (miniscruff) [inline extensions/lifecycle/conductor/service.js:62]: will this config require joi changes?

Comment (naren-rajendran) [reply extensions/lifecycle/conductor/service.js:62]: they all fall under extensions and schema for extensions is just, `joi.object()`

Comment (miniscruff) [reply extensions/lifecycle/conductor/service.js:62]: oh... well that is both good and bad lol

**Review by miniscruff — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

**Pull request merged**